packages/TorneloScoresheet/src/pages/EnterPgn/style.ts

```diff
@@ -1,17 +1,28 @@
-import { StyleSheet } from 'react-native';
+import { Dimensions, StyleSheet } from 'react-native';
+import { TOOLBAR_HEIGHT } from '../../components/Toolbar/style';
 import { colours } from '../../style/colour';
 import { primary } from '../../style/font';
 
+const { height } = Dimensions.get('screen');
+
 export const styles = StyleSheet.create({
   container: {
     display: 'flex',
-    alignItems: 'center',
+    justifyContent: 'space-between',
+    height: height - TOOLBAR_HEIGHT,
```

Comment:
Should we have this in a component like ResponsiveContainer because we'll be setting this height in each View right?

Reply:
Yeah we can definitely look into adding that abstraction. I think this is a fine first iteration though